proto/lbm/wasm/v1/query.proto

```diff
@@ -34,7 +34,7 @@ message QueryInactiveContractsRequest {
 // QueryInactiveContractsResponse is the response type for the
 // Query/InactiveContract RPC method.
 message QueryInactiveContractsResponse {
-  // addresses is the inactive address list
+  // addresses is the inactive address list of byte arrays ordered in ascending order
```

Comment:
isn't it a list of strings?

Reply:
how about 
... list ordered in ascending order of byte formatted addresses

Reply:
This means ascending order when expressed as a byte array, but the type is string.

Reply:
this can be read as
"addresses is the inactive address list of byte arrays _which is_ ordered in ascending order